### PR TITLE
Support for direct setting layout in presenter

### PR DESCRIPTION
**Pull request by besir**

### CODE REVIEW

File: src/Application/UI/Presenter.php [hunk 1]
```diff
@@ -475,7 +475,10 @@ public function findLayoutTemplateFile()
 	{
 		if ($this->layout === FALSE) {
 			return;
+		} elseif (file_exists($this->layout)) {
```
Comment (fprochazka) [inline src/Application/UI/Presenter.php]: I think having some check for `/` or `\`, before the IO call would be faster, than always doing a `file_exists`.

**Comment by fprochazka** [comment]
It's not a bad idea, although I never needed it myself. A test case would be nice :)

**Comment by besir** [comment]
Ok, I will check first letter of the string and after check file_exists(), I dont have Idea for test :-/ THis method have not any test. Can U help me @fprochazka? THX

**Comment by besir** [comment]
```
if ($this->layout === FALSE) {
			return;
		} elseif (in_array(substr($this->layout, 0, 1), ['/', '\\']) && file_exists($this->layout)) {
			return $this->layout;
		}
```
??

**Comment by dg** [comment]
` } elseif (preg_match('#[/\\\\]#', $this->layout) && file_exists($this->layout)) {`

**Comment by besir** [comment]
Why Regexp? Only speed or have any next positives in this context? THX David.

http://www.knesl.com/articles/view/nepouzivejte-regularni-vyrazy

**Comment by dg** [comment]
It checks whether string contains \, not only in first character.

**Comment by besir** [comment]
Its fact, file path dont must be absolute path... Ok, I will test it and i make test (HUH) :D . Thx for cooperatinon.

**Comment by dg** [comment]
Merged e2c1a1da7e3451744c70593f8f38c937cb570a3b

**Pull request closed without merging**